File: isegoria/src/main/java/com/eulersbridge/isegoria/models/PollOption.java
package com.eulersbridge.isegoria.models;

import android.support.annotation.Nullable;

import com.squareup.moshi.Json;

import org.parceler.Transient;

@org.parceler.Parcel
public class PollOption {

    public long id;

    @Json(name = "txt")
    public String text;

    public Photo photo;

    @Json(name = "numOfVoters")
    @Nullable
    public Long votersCount;

    @Json(name = "voted")
    public boolean hasVoted;

    @Transient
    private transient PollResult result;

    public void setResult(PollResult result) {
        this.result = result;
    }

    public PollResult getResult() {
        return result;
    }
}

File: isegoria/src/main/java/com/eulersbridge/isegoria/network/api/models/PollOption.kt
package com.eulersbridge.isegoria.network.api.models

import android.annotation.SuppressLint
import android.os.Parcelable
import com.squareup.moshi.Json
import kotlinx.android.parcel.Parcelize

@SuppressLint("ParcelCreator")
@Parcelize
data class PollOption (

    val id: Long,

    @Json(name = "txt")
    val text: String?,

    val photo: Photo?,

    @Json(name = "numOfVoters")
    val votersCount: Long?,

    @Json(name = "voted")
    var hasVoted: Boolean,

    @Transient
    var result: PollResult?
) : Parcelable